### ISSUE TITLE
Improve `settings.md` wording

### ISSUE THREAD

**Issue by Kludex** [Kludex/uvicorn#1526]
Nice addition!

I think we also need to change the introduction paragraphs in [Settings](https://www.uvicorn.org/settings/):

```markdown
Use the following options to configure Uvicorn, when running from the command line.

If you're running programmatically, using uvicorn.run(...), then use equivalent keyword arguments, eg. uvicorn.run("example:app", port=5000, reload=True, access_log=False). Please note that in this case, if you use reload=True or workers=NUM, you should put uvicorn.run into if __name__ == '__main__' clause in the main module.

You can also configure Uvicorn using environment variables with the prefix UVICORN_. For example, in case you want to run the app on port 5000, just set the environment variable UVICORN_PORT to 5000.

!!! note
    CLI options and the arguments for uvicorn.run() take precedence over environment variables.

    Also note that UVICORN_* prefixed settings cannot be used from within an environment configuration file. Using an environment configuration file with the --env-file flag is intended for configuring the ASGI application that uvicorn runs, rather than configuring uvicorn itself.
```

Suggestion ⬇️

```markdown
Use the following options to configure Uvicorn, when running from the command line.

These options are mirrored as keyword arguments when [running programmatically](/index.md#running-programmatically).

Default settings can also be passed as environment variables using the `UVICORN_` prefix. For example, setting `UVICORN_PORT` to `5000` would make Uvicorn serve on port 5000 if `--port` (when using the CLI) or `port` (when running programmatically) have not been passed.
```

_Originally posted by @florimondmanca in https://github.com/encode/uvicorn/pull/1525#pullrequestreview-1011492897_

**Comment by Kludex** [comment]
I think we should mention who takes precedence as well 🤔

**Comment by iudeen** [comment]
A simple visual representation of what takes precedence would give developers a quick understanding. Since we have multiple levels of precedence, writing would become too wordy 🤔

**Comment by Kludex** [comment]
Closing this for house keeping.
